modulemd/v2/modulemd-defaults.c

```diff
@@ -44,6 +51,28 @@ enum
 static GParamSpec *properties[N_PROPS];
 
 
+gboolean 
+modulemd_defaults_equals(ModulemdDefaults *self_1, ModulemdDefaults *self_2, GError **error)
+{
+  if (!self_2 || !self_1)
```

Comment:
If they're both NULL, we should probably return TRUE, since they do match. If only one or the other is NULL, then this is correct.